test/hotspot/jtreg/runtime/ErrorHandling/HsErrFileUtils.java

```diff
@@ -63,12 +63,10 @@ public static File openHsErrFileFromOutput(OutputAnalyzer output) {
      * if patterns are missing, or if the END marker is missing.
      * @param f Input file
      * @param patterns An array of patterns that need to match, in that order
-     * @param verbose If true, the content of the hs-err file is printed while matching. If false, only the matched patterns
-     *                are printed.
      * @throws RuntimeException, {@link IOException}
```

Comment:
The Javadoc of parameter "verbose" should not be removed here.

Reply:
okay I added it again; but it was already not documented in the same way before my change, once "the matched patterns are printed" once "only important information are printed" . Should we adjust it and to what ?

Reply:
> Looking at what "verbose" does, I'd say the correct description would be "only the matched patterns are printed"

Mostly correct, but there could be also "Lazily skipping the rest of the hs-err file..."  for example; but mostly (almost all) output is about the patterns ("Matches ..." or "Forbidden pattern found  ..." ).